fex2/preconfigured/experiments/common.sh

```diff
@@ -237,26 +258,31 @@ function execute_experiment() {
     fi
 
     if [ -f $BUILD_LOG ]; then
-        if [ "$FORCE_OUTPUT_OVERWRITE" -eq 1 ]; then
+        if is "$FORCE_OUTPUT_OVERWRITE"; then
             echo "" >$BUILD_LOG
         else
             error_exit "$BUILD_LOG already exists. Exiting to avoid corruption. Add '-f' to ignore this error." 1
         fi
     fi
     if [ -f $EXPERIMENT_OUTPUT ]; then
-        if [ "$FORCE_OUTPUT_OVERWRITE" -eq 1 ]; then
+        if is "$FORCE_OUTPUT_OVERWRITE"; then
             echo "" >$EXPERIMENT_OUTPUT
         else
             error_exit "$EXPERIMENT_OUTPUT already exists. Exiting to avoid corruption. Add '-f' to ignore this error." 1
         fi
     fi
     touch $BUILD_LOG $EXPERIMENT_OUTPUT
 
+    echo "[FEX2_HEADER] name: $NAME; experiment_type: $EXPERIMENT_TYPE;" >>"$EXPERIMENT_OUTPUT"
     setup
-    if [ "$NO_BUILD" -ne 1 ]; then
+    if is "$NO_BUILD"; then
+        debug No build
```

Comment:
Shouldn't the argument be a string?

Reply:
I think its already a string this way :thinking: Thought quoting sounds like a good idea :+1: